pkg/kubectl/cmd/apply/apply.go

```diff
@@ -115,6 +115,9 @@ var (
 		# Apply the configuration in pod.json to a pod.
 		kubectl apply -f ./pod.json
 
+        # Apply resources from a directory containing kustomization.yaml
+        kubectl apply -k dir
```

Comment:
make indent consistent (applies to all examples)

Reply:
will update them